docs/notebooks/Common_Gotchas_in_JAX.ipynb

```diff
@@ -568,6 +568,209 @@
     "Note also that, as the two behaviors described above are not inverses of each other, reverse-mode automatic differentiation (which turns index updates into index retrievals and vice versa) [will not preserve the semantics of out of bounds indexing](https://github.com/google/jax/issues/5760). Thus it may be a good idea to think of out-of-bounds indexing in JAX as a case of [undefined behavior](https://en.wikipedia.org/wiki/Undefined_behavior)."
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "LwB07Kx5sgHu"
+   },
+   "source": [
+    "## 🔪 Non-array inputs\n",
```

Comment:
Useful (but maybe not necessary?) suggestion:

- The JAX Sharp Bits docs is getting longer, so this important "sharp" section may be easy to skip when scanning, especially if you're a non-technical user - "attention is all your need"

```suggestion
    "## 🔪 Non-array inputs: NumPy vs. JAX\n",
```

Reply:
Done, thanks!